src/devices/src/virtio/block/io/async_io.rs

```diff
@@ -173,9 +173,9 @@ impl<T> AsyncFileEngine<T> {
 
     #[cfg(test)]
     // Useful when testing, to skip the eventfd polling and waiting.
-    pub fn kick_submission_queue_and_wait(&mut self, min_complete: u32) -> Result<(), Error> {
+    pub fn kick_submission_queue_and_wait_all(&mut self) -> Result<(), Error> {
         self.ring
-            .submit_and_wait(min_complete)
+            .submit_and_wait_all()
             .map(|_| ())
             .map_err(Error::IoUring)
     }
```

Comment:
I think we should move all test-related functions at the end of the file, for improved visibility.